packages/analytics-remote-config/src/remote-config.ts

```diff
@@ -57,7 +57,7 @@ export class RemoteConfigFetch<RemoteConfig extends { [key: string]: object }>
       // Another option is to empty the db if current session doesn't match lastFetchedSessionId
       if (!!lastFetchedSessionId && !!sessionId && lastFetchedSessionId === sessionId) {
         const idbRemoteConfig = await this.remoteConfigIDBStore.getRemoteConfig(configNamespace, key);
-        this.fetchTime = Date.now() - fetchStartTime;
+        this.metrics.set('remote_config_fetch_time_IDB', Date.now() - fetchStartTime);
```

Comment:
In nova, we just take the string as `metric_name` https://github.com/amplitude/nova/blob/01659d95b30f9edaa7372c994d0507e6ece6c157/src/main/java/com/amplitude/event/RequestMetadata.java#L79-L82. I don't think we need to use custom type or enum as it's only used here. Anywhere it's used and could possibly cause inconsistency?

Reply:
I think I prefer the enum or type here since while we probably won't use these again, it'll at least have a single place in this package where we define metric names. Ideally, we'd probably want all the metric names to use roughly the same formatting and having them all defined in one location helps with that.

Reply:
It makes sense. Thank you Lew, I will change it later.